src/Core/Traits/HasMeta.php

```diff
@@ -189,7 +189,7 @@ public function syncMeta(array $post)
         $updates = [];
         $deletes = [];
 
```

Comment:
Please add the `configClass` as part of the trait. That way it always exists on a class, even if it's empty. 

```suggestion
protected string $configClass = null;
```

Reply:
I will go through your suggestions tomorrow, and I am gonna accept them all, except for this one: one cannot declare a variable in trait and then change it in the class using trait: one gets a fatal error.

Reply:
Hm. I was pretty sure that was possible and the manual seems to state it's possible, just that they must be compatible (so both must be `protected string`, etc. After reading that a little closer, though, you're right. It has to have same initial value. Damn. Oh well. :) 

> If a trait defines a property then a class can not define a property with the same name unless it is compatible (same visibility and type, readonly modifier, and initial value), otherwise a fatal error is issued.